jerry-core/parser/js/js-parser.c

```diff
@@ -1439,6 +1441,11 @@ parser_append_breakpoint_info (parser_context_t *context_p, /**< context */
 static ecma_compiled_code_t *
 parser_post_processing (parser_context_t *context_p) /**< context */
 {
+  #if defined  JERRY_DEBUGGER || defined  JERRY_ENABLE_LINE_INFO
+  JERRY_INIT_CURRENT_INSTANCE ();
+  #endif
```

Comment:
if I recall correctly, we always start preprocessor directives on the first column